sockeye/inference.py

```diff
@@ -1535,6 +1544,7 @@ def _beam_search(self,
                 if mx.nd.sum(finished).asscalar() > 0:
                     break
             else:
+                finished = mx.nd.sign(finished + (mx.nd.cast(lengths.reshape(-1), 'int32') >= max_output_lengths))
```

Comment:
matter of taste probably, but ```mx.nd.logical_or(finished, lengths.reshape(-1) >= max_output_lengths))``` might be a bit more readable

Reply:
makes sense. I wasn't aware of the logical_or operator.

Reply:
Please check if logical ops are part of mxnet1.2 already

Reply:
logical_or is not part of mxnet1.2 api installed through pip

Reply:
thanks for checking. we should switch to the logical ops only with mxnet 1.3 then. We have a few other places where these would be handy.